src/test/java/guitests/UITest.java

```diff
@@ -44,7 +45,32 @@
     private static final Logger logger = LogManager.getLogger(UITest.class.getName());
 
     protected static final SettableFuture<Stage> STAGE_FUTURE = SettableFuture.create();
-
+    
+    private static final HashMap<Character, KeyCode> chars; 
+    static {
+        chars = new HashMap<Character, KeyCode>();
+        chars.put('~', KeyCode.BACK_QUOTE);
+        chars.put('!', KeyCode.DIGIT1);
+        chars.put('@', KeyCode.DIGIT2);
+        chars.put('#', KeyCode.DIGIT3);
+        chars.put('$', KeyCode.DIGIT4);
+        chars.put('%', KeyCode.DIGIT5);
+        chars.put('^', KeyCode.DIGIT6);
+        chars.put('&', KeyCode.DIGIT7);
+        chars.put('*', KeyCode.DIGIT8);
+        chars.put('(', KeyCode.DIGIT9);
+        chars.put(')', KeyCode.DIGIT0);
+        chars.put('_', KeyCode.MINUS);
+        chars.put('+', KeyCode.EQUALS);
+        chars.put('{', KeyCode.OPEN_BRACKET);
+        chars.put('}', KeyCode.CLOSE_BRACKET);
+        chars.put(':', KeyCode.SEMICOLON);
+        chars.put('"', KeyCode.QUOTE);
+        chars.put('<', KeyCode.COMMA);
+        chars.put('>', KeyCode.PERIOD);
+        chars.put('?', KeyCode.SLASH);
+    }
```

Comment:
It might be better to put this in a private method below, then do `private static final Map<Character, KeyCode> chars = getKeysWhichCannotBeTyped();` (better names required). The name of the field should also be more specific.

The map can also be made immutable before it's returned from the method (Collections.unmodifiableMap).